src/main/java/com/fundation/webservice/model/ConvertHtmlToDoc.java

```diff
@@ -27,8 +27,9 @@
      * @param criteriaConvert holds the source and destination file paths.
      */
     public void convert(CriteriaConvert criteriaConvert) {
+        CriteriaThumbnailImage criteria = (CriteriaThumbnailImage) criteriaConvert;
         try {
-            Process process = Runtime.getRuntime().exec("3rdparty/" + "pandoc/" + "pandoc.exe -o " + criteriaConvert.getDestPath() + " " + criteriaConvert.getSrcPath());
+            Process process = Runtime.getRuntime().exec(Directories.TOOLS_DIR.getDir() + Directories.PANDOC_DIR.getDir() + "pandoc.exe -o " + criteria.getDestPath() + criteria.getName() + criteria.getExt() + " " + criteria.getSrcPath());
```

Comment:
In this line exceeds 120 lines in length please separate them for better code readability.

Reply:
Fixed.